script/index.js

```diff
@@ -0,0 +1,151 @@
+$(document).ready(() => {
+    // Створюю таби
+    $(".tab").click(function() {
+        $(".tab").removeClass("active").eq($(this).index()).addClass("active");
+        $(".tab-item").hide().eq($(this).index()).fadeIn()
+    }).eq(0).addClass("active");
+    
+    let text = "";
+    let lastBtnText = "";
+    
+    const calc = () => {
+        if (operator(lastBtnText)) {
+            $(".window").html(remove());
+        }
+        let result = eval(text); 
+        $(".window").text('');
+        return result;
+    }
+
+    const remove = () => {
+        let arr = [];
+        arr = text.split("");
+        arr.pop();
+        text = arr.join().replace(/,/g, '');
+        return text;
+    }
+
+    const operator = (oper) => (oper == "*") || (oper == "+") || (oper == "-") || (oper =="/");
+
+    $(".btn").click(function() {
+        $(".result").empty();
+        if (operator(lastBtnText) && operator($(this).text())) {
+            $(".window").html(remove());
+        }
+        $(".window").append(`${$(this).text()}`);
+        lastBtnText = $(this).text();
+        text = $(".window").text(); 
+    });
+
+    $(".btn-equal").click(function() {
+        $(".window").text('');
+        $(".result").html(calc());
+    });
+
+    $(".btn-clear").click(function() {
+        $(".window").text('');
+        $(".result").empty();
+    });
+
+    $(".btn-remove").click(function() {
+        $(".window").html(remove());
+        $(".result").empty();
+    });
+
+    // Slick slider
+
+    $('.variable-width').slick({
+        dots: true,
+        infinite: true,
+        speed: 300,
+        slidesToShow: 1,
+        centerMode: true,
+        variableWidth: true
+    });
+
+    // AJAX:get
+
+    const API = 'https://jsonplaceholder.typicode.com';
+
+    fetch(`${API}/users`)
+    .then(response => response.json())
+    .then(users => {
+        const container = document.getElementsByClassName('container')[0];
+        const table = document.createElement('TABLE');
+        const thead = document.createElement('THEAD');
+        const tbody = document.createElement('TBODY');
+        const userThemes = document.createElement('TR'); 
+        container.appendChild(table);
+        table.appendChild(thead);
+        table.appendChild(tbody);
+        thead.appendChild(userThemes);
+        userThemes.innerHTML = "<th>id</th>" + "<th>name</th>" + "<th>username</th>" + "<th>email</th>" + "<th>city</th>" + "<th>phone</th>" + "<th>website</th>" + "<th>company</th>";
+        users.forEach(user => tbody.appendChild(getUsers(user)));
+    })
+    .catch(error => console.log(error));
+
+    function getUsers(user) {
+        const userPost = document.createElement('TR');
+        const tdId = document.createElement('TD');
```

Comment:
It's a redundant code. Use functions instead. You repeat each line of code a lot

for example use smth like this

```
const tagTDSource = ['name', 'email', 'city', 'phone'];

const createHTMLTags = () => tagTDSource.map(() => document.createElement('TD'));

const [tdName, tdEmail, tdCity, tdPhone] = createHTMLTags(tagTDSource)


console.log(tdName); console.log(tdEmail
```) etc